pass-client-integration/src/test/java/org/dataconservancy/pass/client/integration/UpdateResourceIT.java

```diff
@@ -159,8 +205,31 @@ public void testUpdateWithChange() throws Exception {
         assertNotEquals(deposit.getVersionTag(), updated.getVersionTag());
         assertEquals(expectedStatusRef, updated.getDepositStatusRef());
         assertEquals(deposit.getId().toString(), updated.getId().toString());
+        createdUris.put(deposit.getId(), Deposit.class);
     }
 
+
+    /** simple update test using PUT **/
+    @Test
+    public void testPutUpdateWithChange() throws Exception {
+        //create an instance that uses the PUT update
+        final PassClient overwriteOnUpdateClient = PassClientFactory.getPassClient(true);
+        
+        Deposit deposit = overwriteOnUpdateClient.readResource(overwriteOnUpdateClient.createResource(random(Deposit.class, 1)), Deposit.class);
+
+        String expectedStatusRef = "http://example.org/status/1";
+        deposit.setDepositStatusRef(expectedStatusRef);
+
+        Deposit updated = overwriteOnUpdateClient.updateAndReadResource(deposit, Deposit.class);
+
+        // Doesn't really matter, it changes anyway
+        assertNotEquals(deposit.getVersionTag(), updated.getVersionTag());
+        assertEquals(expectedStatusRef, updated.getDepositStatusRef());
+        assertEquals(deposit.getId().toString(), updated.getId().toString());
+        createdUris.put(deposit.getId(), Deposit.class);
```

Comment:
minor point This should probably be moved earlier, otherwise the deposit might not be cleaned up, and tests down the line might noisily fail if any of the assertions fail